<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResizeRecipe</class>
 <widget class="QDialog" name="ResizeRecipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resize/Convert Recipe</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../SB_Images.qrc">
    <normaloff>:/menu/sb2</normaloff>:/menu/sb2</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="resizeGroup">
     <property name="title">
      <string>Resize Recipe</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="volLabel">
        <property name="text">
         <string>New Size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="volSizeSpin"/>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="volUnitsCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="convertGroup">
     <property name="title">
      <string>Convert Units</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="maltCheck">
        <property name="text">
         <string>Convert Malt</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="maltUnitCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="hopCheck">
        <property name="text">
         <string>Convert Hops</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="hopUnitCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>resizeGroup</zorder>
  <zorder>buttonBox</zorder>
  <zorder>convertGroup</zorder>
 </widget>
 <resources>
  <include location="../SB_Images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ResizeRecipe</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ResizeRecipe</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
